components/stepper/stories/stepper.test.js

```diff
@@ -1,31 +1,69 @@
 import { Variants } from "@spectrum-css/preview/decorators";
 import { Template } from "./template.js";
 
-export const StepperGroup = Variants({
+export const NumberFieldGroup = Variants({
 	Template,
 	testData: [
 		{
 			testHeading: "Default",
 		},
 		{
-			testHeading: "Quiet",
-			isQuiet: true,
+			testHeading: "Hidden stepper",
+			hideStepper: true,
+		},
+		{
+			testHeading: "With help text",
+			helpText: "Help text is here to assist",
+		},
+		{
+			testHeading: "Side label",
+			labelPosition: "side",
+		},
+		{
+			testHeading: "Side label with help text",
+			labelPosition: "side",
+			helpText: "Help text is here to assist",
 		},
 		{
 			testHeading: "Invalid",
 			isInvalid: true,
+			helpText: "Help text is here to assist",
 		},
 		{
-			testHeading: "Quiet + invalid",
-			isQuiet: true,
+			testHeading: "Invalid hidden stepper",
 			isInvalid: true,
+			hideStepper: true,
+			helpText: "Help text is here to assist",
+		},
+		{
+			testHeading: "Minimum width (100px)",
+			wrapperStyles: {
+				"inline-size": "100px",
+			},
 		},
-	],
-	stateData: [
 		{
-			testHeading: "Hide stepper",
+			testHeading: "Default + truncation",
+			withStates: false,
+			value: "1234567890123456789012345678901",
+		},
+		{
+			testHeading: "Hidden Stepper + truncation",
+			withStates: false,
 			hideStepper: true,
+			value: "1234567890123456789012345678901",
 		},
+		{
+			testHeading: "Side label + truncation",
+			withStates: false,
+			labelPosition: "side",
+			value: "1234567890123456789012345678901",
+		},
+		// {
+		// 	testHeading: "Internationalization (Thai)",
+		// 	value: "๐๑๒๓๔๕๖๗๘๙๑๐",
+		// }
```

Comment:
Oh! I really wanted to include an internationalization test to the grid (specifically with all of our bug fixes for Thai in textfield/picker/action button), but I couldn't get the Thai numerals to display. Maybe I don't actually have the right characters?